pkg/subctl/cmd/validate_fwconfig.go

```diff
@@ -0,0 +1,184 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/submariner-io/submariner-operator/apis/submariner/v1alpha1"
+	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/resource"
+)
+
+var validateIntraClusterFWConfig = &cobra.Command{
+	Use:   "fw-intra-cluster",
```

Comment:
How about having `validate fw` as a root, and then `intra-cluster` & `inter-cluster` as sub-commands?
(`inter-cluster` will be for other validation tasks.)
@mangelajo @tpantelis  - What do you think?

Reply:
I'm fine with that if its Okay for others :) 
In that way we can even extend it in future. For example
`subctl validate fw <sub-command>` 

Possible sub-commands
`vxlan` - to validate vxLan firewall configuration in the cluster
`IPsec` - to validate if GatewayNode is configured to allow UDP/4500 and UDP/500
`metrics` - to validate if GatewayNode is configured to allow TCP/8080
etc...

Reply:
Sounds good.  I would suggest `subctl validate firewall ...` so it's self-documenting.